boot/freeldr/freeldr/arch/i386/xboxvideo.c

```diff
@@ -122,48 +120,37 @@ XboxVideoPutChar(int Ch, UCHAR Attr, unsigned X, unsigned Y)
   XboxVideoOutputChar(Ch, X, Y, FgColor, BgColor);
 }
 
+UCHAR
+NvGetCrtc(UCHAR Index)
+{
+  *((PUCHAR) NV2A_CRTC_REGISTER_INDEX) = Index;
```

Comment:
For new code, please use our standard code style indentation (4 spaces).

Reply:
Done.